include/vinecopulib/misc/implementation/tools_stats.ipp

```diff
@@ -140,6 +142,49 @@ to_pseudo_obs_1d(Eigen::VectorXd x,
   return x.array() / (static_cast<double>(n) + 1.0);
 }
 
+//! @param ties_method Indicates how to treat ties; same as in R, see
+//! https://stat.ethz.ch/R-manual/R-devel/library/base/html/rank.html.
+//! @return Pseudo-observations of the copula, i.e. \f$ F_X(x) \f$
+//! (column-wise).
+inline Eigen::MatrixXd
```

Comment:
After thinking about it, shouldn't this belong to wdm since you moved all the related code there?

Reply:
right this function is redundant now, removed it.